### PR TITLE
beansdb: add plist_options and a test

### PR DESCRIPTION
**Pull request by Moisan**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

The plist_options part is related to issue  [#4029](https://github.com/Homebrew/homebrew-core/issues/4029).

### CODE REVIEW

File: Formula/beansdb.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Beansdb < Formula
   homepage "https://github.com/douban/beansdb"
   url "https://github.com/douban/beansdb/archive/v0.6.tar.gz"
   sha256 "b24512862f948d5191f5c43316a41f632bc386f43dcbb69b03ffffe95122a33e"
+  revision 1
```
Comment (MikeMcQuaid) [inline Formula/beansdb.rb]: Don't need a `revision` for this as we're not altering the package contents. `revision` is used to force an upgrade for all users so is only really needed for package contents changes (and even then only really widespread bug fixes).

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by Moisan** [comment]
Testbot failed on `brew doctor`.

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**